includes/class-sensei-grading.php

```diff
@@ -1156,6 +1156,44 @@ public function get_graded_lessons_average_grade() {
 		return $average_grade_value;
 	}
 
+	/**
+	 * Get average grade of all lessons graded in all the courses filtered by students.
+	 *
+	 * @since 4.5.0
+	 * @access public
+	 *
+	 * @deprecated 4.5.0 use Sensei_Reports_Overview_Service_Students::get_graded_lessons_average_grade_filter_users
+	 *
+	 * @param array $user_ids user ids.
+	 * @return double $graded_lesson_average_grade Average value of all the graded lessons in all the courses.
+	 */
+	public function get_graded_lessons_average_grade_filter_users( $user_ids ) {
```

Comment:
This method could be removed now, right?

Reply:
Yep, you can just remove it because it's not released yet.

Reply:
removed: https://github.com/Automattic/sensei/pull/5091/commits/720bd0e24db0d18e6babad8efc6ec19fce4ebd16